Dockerfile-local

```diff
@@ -0,0 +1,6 @@
+# Use distroless as minimal base image to package the manager binary
+# Refer to https://github.com/GoogleContainerTools/distroless for more details
+FROM gcr.io/distroless/static:latest
```

Comment:
Do we need another Dockerfile?

Reply:
This dockerfile is for local build testing purposes. mac m1 over Rancher had some problems to build amd64 image thru qemu, this is to move build stage to laptop and copy binary directly from laptop.